packages/lib-classifier/src/store/FieldGuideStore.spec.js

```diff
@@ -48,35 +47,47 @@ describe('Model > FieldGuideStore', function () {
       })
   }
 
+  function setupStores (clientStub) {
+    rootStore = RootStore.create({
+      classifications: {},
+      dataVisAnnotating: {},
+      drawing: {},
+      feedback: {},
+      subjects: {},
+      subjectViewer: {},
+      tutorials: {},
+      workflows: {},
+      workflowSteps: {},
+      userProjectPreferences: {}
+    }, { client: clientStub })
+  }
+
+  afterEach(function () {
+    rootStore = null
```

Comment:
This can be removed too, in favour of declaring a new mock store for each test block.